### PR TITLE
added title for Y axis when there is single series

### PR DESCRIPTION
**Pull request by anuveyatsu**
Also updated tests for view.js and snapshot for MultiViews component.

### CODE REVIEW

File: src/utils/view.js [hunk 1]
```diff
@@ -61,6 +61,11 @@ export function simpleToPlotly(view) {
       }
     }
   }
+  if(view.spec.series.length === 1) {
```
Comment (rufuspollock) [inline src/utils/view.js:64]: What about case where we have multiple series: are we setting a legend on the diagram as per discussion?

**Comment by rufuspollock** [comment]
@anuveyatsu commit is straight forward and looks good. One question as above is about multiple series case.

File: src/utils/view.js (hunk 1 above)
Comment (anuveyatsu) [reply src/utils/view.js:64]: @rufuspollock Fortunately, Plotly handles it by default:
image

Comment (rufuspollock) [reply src/utils/view.js:64]: @anuveyatsu 😄

**Pull request merged**